docs/api/setup-server/listen.mdx

```diff
@@ -16,12 +16,15 @@ Specifies how to handle a request that is not listed in the request handlers.
 
 #### Pre-defined behaviors
 
+With these options, the unhandled request gets performed and a message is logged. If you want to prevent the request altogether, you can use a custom callback.
+
 | Option name | Description                                            |
 | ----------- | ------------------------------------------------------ |
-| `"bypass"`  | Performs an unhandled request as-is.                   |
+| `"bypass"`  | Does not print anything.                   |
```

Comment:
This is a great point. The behavior is somewhat different from what you're describing here because the "error" option _doesn't_ perform the request.

I suggest rephrasing the option descriptions to:

- "bypass" performs the request but prints nothing if it has no handler.
- "warn" performs the request and prints a warning if it has no handler.
- "error" _does not_ perform the request and throws an exception if it has no handler.

What do you think about this suggestion?